charts/opentelemetry-demo/values.yaml

```diff
@@ -70,82 +70,82 @@ components:
     enabled: true
     servicePort: 8080
     env:
-    - name: PORT
-      value: '8080'
+      - name: PORT
+        value: "8080"
     podAnnotations: {}
     #  instrumentation.opentelemetry.io/inject-sdk: "true"
 
   emailService:
     enabled: true
     servicePort: 8080
     env:
-    - name: APP_ENV
-      value: production
-    - name: PORT
-      value: '8080'
+      - name: APP_ENV
+        value: production
+      - name: PORT
+        value: "8080"
     podAnnotations: {}
     #  instrumentation.opentelemetry.io/inject-sdk: "true"
 
   featureflagService:
     enabled: true
     env:
-    # DATABASE_URL will be automatically added to use the enabled ffsPostgres component.
-    # If a different url is desired, set DATABASE_URL here.
-    # - name: DATABASE_URL
-    #   value:
-    - name: FEATURE_FLAG_GRPC_SERVICE_PORT
-      value: '50053'
-    - name: FEATURE_FLAG_SERVICE_PORT
-      value: '8081'
+      # DATABASE_URL will be automatically added to use the enabled ffsPostgres component.
+      # If a different url is desired, set DATABASE_URL here.
+      # - name: DATABASE_URL
+      #   value:
+      - name: FEATURE_FLAG_GRPC_SERVICE_PORT
+        value: "50053"
+      - name: FEATURE_FLAG_SERVICE_PORT
+        value: "8081"
     ports:
-    - name: grpc
-      value: 50053
-    - name: http
-      value: 8081
+      - name: grpc
+        value: 50053
+      - name: http
+        value: 8081
```

Comment:
I like this formatting as well. Can you submit another PR for that?

Reply:
Will do

Reply:
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/392/files